mustreams/src/index.ts

```diff
@@ -120,6 +120,18 @@ export class MuWriteStream {
         this.offset += 8;
     }
 
+    public writeASCII (str:string) {
+        const strLeng = str.length;
+        const bytes = new Array(strLeng);
+        for (let i = 0; i < strLeng; ++i) {
```

Comment:
We could directly write the bytes to stream using `writeUint8`  This saves allocating an array, and should be faster.